DigitalOffice/Services/TimeManagementService/Mappers/Interfaces/IMapper.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace LT.DigitalOffice.TimeManagementService.Mappers.Interfaces
+{
+    public interface IMapper<Tin, Tout>
+    {
+        Tout Map(Tin value);
+    }
```

Comment:
All generics usually (for example, Task that returns "TResult" in Microsoft documentation) are written like TSomething, 
where "Something" starts with an Upper letter. I think we should write everywhere TIn, TOut etc.

Reply:
Fixed.